src/main/scala/io/iohk/ethereum/Test.scala

```diff
@@ -0,0 +1,31 @@
+package io.iohk.ethereum
```

Comment:
I'm not sure whether we should incorporate this to the codebase, if needed maybe we could create an unit test that tests the same?

Reply:
We shouldn't, it's just an internal test :) I'll exclude the file in the next commit.